src/main/java/edu/cornell/kfs/gl/batch/service/impl/CuPosterServiceImpl.java

```diff
@@ -52,4 +60,161 @@ protected KualiDecimal getPercentage(KualiDecimal amount, BigDecimal percent) {
         return new KualiDecimal(result);
     }
 
+    /**
+     * CUMod to prefix the "Indirect Cost Recovery System Generated" edoc number with "ICR"
+     * 
+     * Generate a transfer transaction and an offset transaction
+     *
+     * @param et                         an expenditure transaction
+     * @param generatedTransactionAmount the amount of the transaction
+     * @param runDate                    the transaction date for the newly created origin entry
+     * @param group                      the group to save the origin entry to
+     */
+    protected final static String ICR_EDOC_PREFIX = "ICR";
```

Comment:
I would suggest moving this constant towards the beginning of the class to make it easier to find, and to also reorder the declaration to be "static final" instead of "final static".

Reply:
My understanding was that this fix was going to be suggested as a KualiCo mod. Therefore, my intention of putting the constant here was to keep it close to the actual method where it was being used so it was easier to find during packaging of that work.

Reply:
I'll flip the "static final".

Reply:
Ok, keeping the constant in this position is okay with me, then. And thanks for moving forward with the "static final" flip.